tests/pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+addopts = --strict-markers
```

Comment:
tests/pytest.ini  file is OK ?  
Or should I put it into tox.ini into [pytest] section ?

Reply:
You shouldn't touch `pytest.ini` but let double-check with @ahopkins .

Reply:
--strict-markers should be our friend - at least i think so - but I may be wrong.

Reply:
@ahopkins we need your input here.

Reply:
If you are making a change to it, it should be in `tox`, and I would prefer that be a separate PR.

Reply:
OK.
Please look:  #1923